src/worker/queue.ts

```diff
@@ -39,5 +41,5 @@ export function startWorker(server: PluginsServer): Worker {
 
     worker.start()
 
-    return worker
+    return async () => worker.stop()
```

Comment:
Why not just return the whole worker for more flexibility. Could then call `worker.stop` ourselves somewhere else, but also any other `worker` method.

Reply:
Indeed, why not.
Historically the old `queue` ran both `worker.stop()` and `client.stop()` on shutdown, yet now that celery is inlined and there's no separate connection being opened in `client`, we can indeed directly return the `worker`. Fixed